prod-devel-rcar.yaml

```diff
@@ -204,7 +204,7 @@ components:
       - *COMMON_SOURCES
       - type: git
         url: https://github.com/renesas-rcar/meta-renesas.git
-        rev: "kirkstone-dev"
+        rev: "644ccf213fed7be3cf68b2a7bd80ae32bcc8a3c8"
```

Comment:
Why are you doing this? Idea of our master branch is to closely follow Renesas branches, so we can fix issues as soon as they appear

Reply:
I understand your point and agree in general. However I would prefer fixing revs for the time being together with the opening an issue and unfix them in due course.

Reply:
I decided to remove that commit for v2, so no fix revs